python/cuml/common/sparsefuncs.py

```diff
@@ -73,6 +73,23 @@ def _map_l2_norm_kernel(dtype):
     return cuda_kernel_factory(map_kernel_str, dtype, "map_l2_norm_kernel")
 
 
+def _map_mul_sparse_array(dtype):
+    """Creates cupy RawKernel for multiplying sparse array with a dense ar """
+
+    map_kernel_str = r'''
```

Comment:
We should expect indices and indptr will be of the same type so those should share the same number in `{}`. I think this kernel would be better written as an ElementWise kernel, where `i` is used in place of `tid`. In general, we are going to be getting rid of the kernel factory soon.

Reply:
Resolved as we got rid of the kernel with  https://github.com/rapidsai/cuml/commit/1efcbde48a7077c2522e70587d96fe68d6fee11c  .